addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -0,0 +1,147 @@
+from odoo import api, fields, models, _
+from ..misc import date_diff, validate_dates
+
+
+class StockPickingExtraFields(models.Model):
```

Comment:
As this model and extra fields are in the Delivery Control module, I think the model should be named more specifically, i.e. `StockPickingDeliveryControl`

Reply:
done